api4/user.go

```
@@ -581,8 +581,8 @@ func patchUser(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	ouser, err := c.App.GetUser(c.Params.UserId)
```

We shouldn't ignore this error either (in the `else` case).